src/yyjson.h

```diff
@@ -6125,6 +6125,93 @@ yyjson_api_inline yyjson_mut_val *yyjson_mut_doc_get_pointer(
     return yyjson_mut_get_pointer(doc ? doc->root : NULL, ptr);
 }
 
+/*==============================================================================
+ * JSON Value at Pointer API (Implementation)
+ *============================================================================*/
+
+/** Set provided `value` if the JSON Pointer (RFC 6901) exists and is type bool.
+    Returns true if value at `ptr` exists and is the correct type, otherwise false. */
+yyjson_api_inline bool yyjson_get_bool_pointer(
+    yyjson_doc *doc, const char *ptr, bool *value) {
+    yyjson_val *val = yyjson_doc_get_pointer(doc, ptr);
+    if (val && yyjson_is_bool(val)) {
```

Comment:
`yyjson_is_bool(val)` already checks for NULL `val`, so we can skip that. However, we need to make sure the `value` pointer is not NULL.

Reply:
Thanks.  NULL checks for `value` pointers added.